src/main/java/de/themorpheus/edu/taskservice/endpoint/dto/response/solution/GetImageSolutionResponseDTO.java

```diff
@@ -0,0 +1,21 @@
+package de.themorpheus.edu.taskservice.endpoint.dto.response.solution;
```

Comment:
Delete bc unneccessary

Reply:
Same. Why? I need it to return my solution.

Reply:
If you just need it to return getImageSolution delete it 
else don't